pkg/api/types.go

```diff
@@ -719,11 +719,11 @@ type Probe struct {
 	// The action taken to determine the health of a container
 	Handler `json:",inline"`
 	// Length of time before health checking is activated.  In seconds.
-	InitialDelaySeconds int64 `json:"initialDelaySeconds,omitempty"`
+	InitialDelaySeconds int `json:"initialDelaySeconds,omitempty"`
```

Comment:
We should document that in the conventions.  It's obviously easier to use `int` internally.  Will the codegen converters convert int32 -> int ?